app/assets/javascripts/discourse/widgets/embedded-post.js.es6

```diff
@@ -27,6 +28,8 @@ export default createWidget('embedded-post', {
   buildKey: attrs => `embedded-post-${attrs.id}`,
 
   html(attrs, state) {
+    this.model = Post.create(attrs);
```

Comment:
`getAncestorModel` returns the post of the widget that the `embedded-post` widget is attached to.  Therefore, we end up getting the wrong model in cases where we call `helper.getModel()` in `decorateCooked` callbacks.